docker/docker-compose.yaml

```diff
@@ -3,12 +3,25 @@ version: '3.9'
 networks:
   predictive-maintenance-net:
     driver: bridge
+  rabbitnet:
+    name: rabbitnet
+    driver: bridge
 
 volumes:
   prometheus-data:
     driver: local
 
 services:
+  rabbitmq:
+    image: rabbitmq:3.11-management
+    container_name: rabbitmq
+    user: ":"
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+    networks:
+      - rabbitnet
+
```

Comment:
Why is this its own network?

Reply:
Changed to now use the same network as the other containers